python/tank/platform/bundle.py

```diff
@@ -920,6 +935,37 @@ def _get_engine_name(self):
 
         return engine_name
 
+    def _get_metrics_context(self):
+        """
+        Should be re-implemented in deriving classes and return a dictionary with
+        the properties needed to log a metric event for this bundle.
```

Comment:
please document what each implementation is expected to return and great if we can include a concrete example of what is expected.